api/hydrophoneApi.go

```diff
@@ -487,3 +491,36 @@ func (a *Api) isAuthorizedUser(tokenData *token.TokenData, userId string) bool {
 		return false
 	}
 }
+
+func (a *Api) verifySendAttempts(ctx context.Context, confirmationType models.Type, creatorId string, email string, userId string) (bool, int64, error) {
+	confirm := models.Confirmation{
+		Type:      confirmationType,
+		CreatorId: creatorId,
+		Email:     email,
+		UserId:    userId,
+	}
+	createdSince := time.Now().Add(-a.Config.ConfirmationAttemptsTimeWindow)
+	var count int64
+	var err error
+	if confirm.Type == models.TypeSignUp {
+		var res *models.Confirmation
+		res, err = a.Store.FindConfirmation(ctx, &confirm)
+		if err == nil {
+			if res == nil {
+				count = 0
+			} else if res.Created.After(createdSince) {
+				count = res.ResendCounter
+			}
+		}
+	} else {
+		count, err = a.Store.CountLatestConfirmations(ctx, confirm, createdSince)
+		a.logger.Printf("verifySendAttempts::count:%v for %v", count, confirm)
+	}
+	if err != nil {
+		return false, 0, err
+	}
+	if count > a.Config.ConfirmationAttempts {
```

Comment:
```suggestion
	if count >= a.Config.ConfirmationAttempts {
```

Reply:
I would have limited the number of attempts to what we specifiy in the configuration.
Not critical but a bit missleading I think